robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBluetoothDevice.java

```diff
@@ -3,12 +3,27 @@
 import android.bluetooth.BluetoothDevice;
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
+import org.robolectric.annotation.RealObject;
+import org.robolectric.util.ReflectionHelpers;
 
 @Implements(BluetoothDevice.class)
 public class ShadowBluetoothDevice {
 
+  @RealObject private BluetoothDevice realObject;
   private String name;
 
+  public void __constructor__(String address) {
+    // Call getService() like the original constructor would, but ignore any Exceptions. This allows
```

Comment:
If this were a javadoc comment it'd automatically appear in the Android developer docs with the Robolectric plugin.